internal/controller/apiextensions/composite/revision.go

```diff
@@ -41,6 +41,7 @@ func AsCompositionSpec(crs v1alpha1.CompositionRevisionSpec) v1.CompositionSpec
 		PatchSets:                         make([]v1.PatchSet, len(crs.PatchSets)),
 		Resources:                         make([]v1.ComposedTemplate, len(crs.Resources)),
 		WriteConnectionSecretsToNamespace: crs.WriteConnectionSecretsToNamespace,
+		PublishConnectionDetailsWithStoreConfigRef: crs.PublishConnectionDetailsWithStoreConfigRef,
```

Comment:
Hrm, this looks like it isn't `gofmt`-ed, somehow. I would have expected the linter to complain.

Reply:
It looks like this is how this is `gofmt`-ed :)
https://user-images.githubusercontent.com/9900707/158379421-0850ed91-9378-4783-bca8-fa9d8be41c1c.mov